nss-day-cohort-60/national-parks-django-api
# Registration modal added to login

Description:

Added registration modal to login modal
Added password field to login modal
Hid favorites and create blog buttons from view when not logged in

Steps to test: 
1. Fetch md-add-modals branch and switch to it
2. Click on login, then register here button. Enter in a new user and verify that it worked.
3. Navigate to blogs page when logged in and logged out and verify that create blog and favorites button are not visible when logged out.
